Documentation/configuration/sbom.rst

```diff
@@ -31,22 +31,4 @@ Prerequisites
 Download SBOM
 =============
 
-The SBOM can be downloaded from the supplied Cilium image using the
-``cosign download sbom`` command.
-
-.. code-block:: shell-session
-
-    $ cosign download sbom --output-file sbom.spdx <Image URL>
-
-Verify SBOM Image Signature
-===========================
-
-To ensure the SBOM is tamper-proof, its signature can be verified using the
-``cosign verify`` command.
-
-.. code-block:: shell-session
-
-    $ COSIGN_EXPERIMENTAL=1 cosign verify --certificate-github-workflow-repository cilium/cilium --certificate-oidc-issuer https://token.actions.githubusercontent.com --attachment sbom <Image URL> | jq
-
-It can be validated that the image was signed using Github Actions in the Cilium
-repository from the ``Issuer`` and ``Subject`` fields of the output.
```

Comment:
Why removing these comments? Do they no longer apply? How does one verify that the SBOM has been correctly signed?

Reply:
I believe the `cosign download attestation` only downloads without verifying if they are correctly signed. Shouldn't we use `cosign verify-attestation` as well?

Reply:
You are right. When `cosign download sbom` command is run, it outputs:

> WARNING: Downloading SBOMs this way does not ensure its authenticity. If you want to ensure a tamper-proof SBOM, download it using 'cosign download attestation <image uri>'.

This got me thinking  `cosign download attestation <image uri>` automatically verifies an attestation on the supplied container image to ensure a tamper-proof SBOM.  We should use `cosign verify-attestation` as well.